.gitignore

```diff
@@ -1,9 +1,5 @@
 node_modules
 
-/server-cert.pem
-
-/server-key.pem
-
```

Comment:
I just noticed these are going into a different directory. Perhaps we should just .gitignore the directory in this file instead so that the folder isn't committed. Is there ever a need for the `certs` dir to be committed?